index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Space Lovers</title>
+  </head>
+  <body>
+    <header>
+      <div class="socials">
+        <ul>
+          <li>
+            <a href="#"><img src="./assets/Linkedin.png" alt="linkedin" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./assets//Twitter.png" alt="twitter" /></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My page</a></li>
+          <li><a href="#">Log out</a></li>
+        </ul>
+      </div>
+      <nav>
+        <div class="logoText">
+          <a href="/">
+            <img src="./assets/logo.png" alt="logo" class="logo" />
+            <h2>EES@</h2>
+          </a>
+        </div>
+        <div class="sm-header">
+          <img src="./assets/menu.png" alt="menu-icon" />
+        </div>
+        <div class="lg-header">
+          <ul>
+            <li><a href="about.html"> About</a></li>
+            <li><a href="#"> Program </a></li>
+            <li><a href="#"> Join</a></li>
+            <li><a href="#"> Sponser</a></li>
+            <li><a href="#"> News</a></li>
+          </ul>
+          <a href="#">
+            <p>Space Lovers</p>
+          </a>
+        </div>
+      </nav>
+    </header>
+    <nav class="menuContainer">
+      <div class="closeButton">
+        <img src="./assets/Icon-Cancel.png" alt="close" />
+      </div>
+      <ul>
+        <li><a href="about.html"> About</a></li>
+        <li><a href="#"> Program </a></li>
+        <li><a href="#"> Join</a></li>
+        <li><a href="#"> Sponser</a></li>
+        <li><a href="#"> News</a></li>
+      </ul>
+    </nav>
+    <main class="container">
+      <section class="section-1">
+        <h2>"Hello! Space Lovers"</h2>
+        <h1>Global Space Events 2021</h1>
+        <div class="intro">
+          <p>
+            Space exploration is the use of astronomy and space technology to
+            explore outer space. While the exploration of space is carried out
+            mainly by astronomers with telescopes, its physical exploration
+            though is conducted both by unmanned robotic space probes and human
+            spaceflight
+          </p>
+          <div class="timing">2021.11.16(TUE) ~ 19(FRI)</div>
+          <div class="location">@ National Space Community</div>
+        </div>
+      </section>
```

Comment:
- [ ] As stated by the previous reviewer, Kindly make sure this section matches the proper left and right margins, alignments font-size and font decoration of the original design😎✌

Reply:
please I don't know what the problem is now I fixed it in the last commit so could you clarify it for me @rotimiazeez